packages/protocol/contracts/governance/Validators.sol

```diff
@@ -1131,4 +1132,30 @@ contract Validators is
       }
     }
   }
+
+  /**
+  * @notice Returns the group that `account` was a member of during `epochNumber`.
+  * @param account The account whose group membership should be returned.
+  * @param epochNumber The epoch number we are querying this account's membership at.
+  * @param index The index into the validator's history struct for their history at `epochNumber`.
+  * @return The group that `account` was a member of during `epochNumber`.
+  */
+  function groupMembershipInEpoch(address account, uint256 epochNumber, uint256 index)
+    external
+    view
+    onlyRegisteredContracts(canForceDeaffiliation)
```

Comment:
Oops :)

Reply:
I assume this was an edit?

Reply:
This is a view call, it should be fine for any address to call, so I don't think we need this modifier